internal/util/porch/client.go

```diff
@@ -34,7 +36,10 @@ func CreateClient(flags *genericclioptions.ConfigFlags) (client.Client, error) {
 		return nil, err
 	}
 
-	c, err := client.New(config, client.Options{Scheme: scheme})
+	c, err := client.New(config, client.Options{
+		Scheme: scheme,
+		Mapper: createRESTMapper(),
```

Comment:
I didn't see how this is used (and it doesn't appear to be test code).  Am I missing it?

Reply:
When I was building the test against the fake server the go-client would by default interrogate the server openapi specs to construct the rest mapping. So to avoid having to fake all of the openapi methods I provided own rest mapper for the types that the CLI uses.

So far I haven't found a better way :(